src/Stripe.net/Services/ExternalAccounts/ExternalAccountUpdateOptions.cs

```diff
@@ -12,6 +13,7 @@ public class ExternalAccountUpdateOptions : BaseOptions, IHasMetadata
         public bool? DefaultForCurrency { get; set; }
 
         [JsonProperty("external_account")]
+        [AllowNameMismatch]
         public ExternalAccountCardUpdateOptions ExternalAccountCard { get; set; }
```

Comment:
ooc couldn't you use your new magic thing that takes N parameters for the same property name instead?

Reply:
Nah, it doesn't work like that. `AnyOf<>` lets you wrap multiple types, not multiple names.

Reply:
Sure I know. I'm saying we should remove `ExternalAccountCard` and `ExternalAccountBankAccount` and use `AnyOf` + `ExternalAccount` no?

Reply:
Looks like `ExternalAccountCard` never worked anyway. Fixed in #1806.